app/lib/notifications.server.ts

```diff
@@ -46,4 +51,10 @@ const sendNotification = async (emailAddress: string, subject: string, text: str
   }
 };
 
+export async function init() {
+  logger.debug('Notifications init: adding jobs for certificate/record expiration notices');
+  await addCertificateExpirationNotifications();
+  await addRecordExpirationNotifications();
```

Comment:
Actually I don't know that there is, this looks good.

Reply:
It's helpful to know that background jobs are being added automatically via the log message, as it will happen on startup without any UI interaction starting the process.